scipy/stats/morestats.py

```diff
@@ -206,15 +204,42 @@ def kstat(data, n=2):
 
     http://mathworld.wolfram.com/Cumulant.html
 
+    Examples
+    --------
+    >>> from scipy import stats
+    >>> rndm = np.random.RandomState(1234)
+
+    As sample size increases, n-th moment and n-th k-statistic converge to the
+    same number (although they aren't identical)
```

Comment:
... and this "same number" is zero for the normal distribution, even though the convergence is not very fast.